test/unit/annotation_spec.js

```diff
@@ -1659,6 +1685,64 @@ describe("annotation", function () {
         done();
       }, done.fail);
     });
+
```

Comment:
Let's move this new unit test just above the `"should handle radio buttons with a field value"` test so the checkbox tests are nicely grouped.